src/client/components/ProductsList.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class ProductsList extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      limit: 3
+    };
+  }
+
+  showMore = () => {
+    const { limit } = this.state;
+    this.setState({ limit: limit + 3 });
```

Comment:
These two lines could be replaced with one line:
` this.setState(({ limit }) => ({ limit: limit + 3 }));`